tools/stats/grouping.py

```diff
@@ -151,7 +151,7 @@ def is_new_item(line):
                     rval = round(rval)
                 else:
                     rval = '%g' % rval
-                        
+                rval = int(rval)
```

Comment:
This is not the correct place for this cast. I think it should be in line 151:

`rval = int(round(rval))`